assets/css/style.css

```diff
@@ -283,6 +283,28 @@ html[dir='rtl'] #srcLanguages {
     clear: both;
 }
 
+/* Dictionary Lookup */
+
+#dictionaryLookup {
+    margin-top: 1em;
+}
+
+#dictTranslation {
+    color: #737373;
+}
+
+#currentWordForLookup {
+    color: #000;
+}
+
+#dynamicLocalization {
```

Comment:
This should be more specific about what it actually refers to. Right now it's more of a meta id.